daemon/src/taker.rs

```diff
@@ -172,8 +173,10 @@ async fn main() -> Result<()> {
 
                 let (monitor_actor_address, monitor_actor_context) = xtra::Context::new(None);
 
+                let mut conn = db.acquire().await.unwrap();
+                let cfds = load_all_cfds(&mut conn).await.unwrap();
```

Comment:
If we are loading these out here already, I think we should also pass them into the `taker_cfd::Actor` instead of loading them inside the constructor again!

Reply:
Not sure about this one, the cfd actor explicitly has access to the db and should always reason about the cfds that are in the db. 
If any actor does changes to the cfds in the db this list of cfds will not be up to date anymore. Currently this would work, because we know that there are no changes in between, but I would rather not optimize that. I think the cfd actor, given that it has access to the db, should always load from the db.

Reply:
The thing is, multiple actors (like the monitoring one) need access to the list of CFDs that are initially present in the database upon startup. It seems unnecessary to load these multiple times. Additionally, to reach a consistent system, this should actually be the exact same list for every actor. Passing in a list is also synchronous which is a plus for a constructor IMO.

Reply:
fair enough, after Mariusz' changes merged I changed it because now I don't have the cfds anymore because we have a separate task for populating the feed. 
I still think we have to be careful with this approach though - the only thing that should ever change cfds in the db prior to the actors starting should be cleanup. And that is always run before anything starts up. We should avoid a situation where we pass an outdated list of cfds around.